code/modules/unit_tests/json_savefile_importing.dm

```diff
@@ -0,0 +1,54 @@
+/**
+ * The job of this unit test is to ensure that save files are correctly imported from BYOND to JSON
+ * Its a rather convoluted process and so this test ensures that something didnt fucking up somewhere
+ */
+/datum/unit_test/json_savefiles
```

Comment:
I would also make another test that makes sure that stuff like strings with `null`, `[]`, etc don't for whatever reason parse. I expect them not to but worth testing as a fuzz.